ci-operator/config/openshift/console/openshift-console-master.yaml

```diff
@@ -12,6 +16,9 @@ images:
 - dockerfile_path: Dockerfile
   from: base
   to: console
+- dockerfile_path: frontend/packages/kubevirt-plugin/integration-tests/Dockerfile
+  from: base_centos
+  to: kubevirt_test
```

Comment:
the base image should include the repository for the tests to run, right? Or is there any other configuration required?